Moya.podspec

```diff
@@ -49,4 +49,15 @@ Pod::Spec.new do |s|
     ss.dependency "Moya/Core"
     ss.dependency "RxSwift", "~> 6.0"
   end
+  
+  s.subspec "Async" do |ss|
+    ss.source_files  = "Sources/AsyncMoya/"
+    ss.dependency "Moya/Core"
+    ss.ios.deployment_target = '13.0'
+    ss.osx.deployment_target = '10.15'
+    ss.tvos.deployment_target = '13.0'
+    ss.watchos.deployment_target = '6.0'
+    ss.swift_version = '5.5'
```

Comment:
I ran some tests and your implementation works pretty well in my project where I use Swift Package Manager only.

But I got an error when I try to install it in a project using only cocoapods.
Were you be able to test it using cocoapods with success?

My project uses the latest Swift version and this is how I'm trying to install it:
```ruby
pod 'Moya', :git => 'https://github.com/gre4ixin/Moya.git', :branch => 'development'
pod 'Moya/Async', :git => 'https://github.com/gre4ixin/Moya.git', :branch => 'development'
```

Reply:
This the error: 

<img width="764" alt="Screen Shot 2022-01-09 at 5 40 08 PM" src="https://user-images.githubusercontent.com/12019206/148704061-930988a5-925c-46ea-8f52-f06eee47c7ef.png">

Reply:
Is that line really required?

Reply:
I think that's not required, because in code i use #if #endif with check swift version. I fix this line in podspec now.

Reply:
OK. I'll test it gain when you push your changes

Reply:
@thejohnlima pushed

Reply:
Works